aquifer-fetkovich/eclipse-simulation/2D_FETKOVICHAQUIFER.DATA

```diff
@@ -0,0 +1,648 @@
+-- This reservoir simulation deck is made available under the Open Database

```

Comment:
This file is not needed.

Reply:
Ok! Should I keep the directories? Or can I remove these too?

Reply:
I think you do not need to keep the directories unless you need to have files there.

Reply:
Actually, git does not care about directories, so you do not need to do anything with them. Git only tracks files